sympy/physics/vector/tests/test_point.py

```diff
@@ -76,8 +96,8 @@ def test_point_funcs():
     B = ReferenceFrame('B')
     B.set_ang_vel(N, 5 * B.y)
     O = Point('O')
-    P = O.locatenew('P', q * B.x)
-    assert P.pos_from(O) == q * B.x
+    P = O.locatenew('P', q * B.x + q2 * B.y)
+    assert P.pos_from(O) == q * B.x + q2 * B.y
```

Comment:
Was this an error in the example?

Reply:
Yes, the next line sets the velocity as `P.set_vel(B, qd * B.x + q2d * B.y)`